src/cleanup_multi_level_immediately_unittest.cc

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2024, Google Inc.
```

Comment:
Instead of introducing a new unit test that's almost identical to the original one, existing tests should be extended instead.